raft/handler.go

```diff
@@ -656,11 +675,24 @@ func (pm *ProtocolManager) addPeer(address *Address) {
 	}
 
 	// Add P2P connection:
-	p2pNode := enode.NewV4(pubKey, address.Ip, int(address.P2pPort), 0, int(address.RaftPort))
+	var ip net.IP
+	if ip = net.ParseIP(address.Hostname); ip == nil {
+		// QUORUM
+		// attempt to look up IP addresses if host is a FQDN
+		lookupIPs, err := net.LookupIP(string(address.Hostname))
+		if err != nil {
+			panic(err.Error())
```

Comment:
Node is not able to start. This happens when peers' hostnames and IPs have not yet registered in DNS. 

It is a challenge for environments where  DNS entry is only available when VM starts up.